uppsrc/CtrlCore/X11DHCtrl.cpp

```diff
@@ -197,8 +197,8 @@ bool DHCtrl::Init()
 	    			)
 	    : NULL;
 
-	top = new Top;
-	top->window = hwnd;
+	utop = new Top;
+	utop->window = hwnd;
```

Comment:
You should use SetTop() here
```
auto pTop = newTop;
pTop->window = hwnd;
SetTop(pTop);
```
SetTop also setting top to true, which is missing in this implementation.

Reply:
Good point about the setting top to true.

Reply:
Done.